tests/basic/pure_test.go

```diff
@@ -3505,6 +3505,386 @@ var _ = Describe("{IscsiPortsDownDuringNewPoolCreateInProgress}", func() {
+				ReclaimPolicy:     &reclaimPolicyDelete,
+				VolumeBindingMode: &bindMode,
+				AllowedTopologies: allowedTop,
+			}
+			storage.Instance().DeleteStorageClass(scName)
+			time.Sleep(1 * time.Second)
+			sc, err := storage.Instance().CreateStorageClass(&scObj)
+			log.Infof("Creation of sc[%v] got err[%v]", sc, err)
+			return err
+		}
+		createNS := func(nsName string) error {
+			ns := &v1.Namespace{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: nsName,
+				},
+			}
+			log.InfoD("Create NS %v", nsName)
+			_, err := core.Instance().CreateNamespace(ns)
+			if err != nil {
+				if apierrors.IsAlreadyExists(err) {
+					log.Infof("Namespace %s already exists. Skipping creation.", ns.Name)
+					err = nil
+				} else {
+					log.Infof(fmt.Sprintf("error creating namespace [%s] failed [%v]", nsName, err))
+				}
+			}
+			return err
+		}
+		createPVC := func(pvcName, scName, appNs string) error {
+			log.InfoD("creating PVC [%s] in namespace [%s]", pvcName, appNs)
+
+			pvcObj := &v1.PersistentVolumeClaim{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      pvcName,
+					Namespace: appNs,
+				},
+				Spec: v1.PersistentVolumeClaimSpec{
+					AccessModes:      []v1.PersistentVolumeAccessMode{v1.ReadWriteOnce},
+					StorageClassName: &scName,
+					Resources: v1.ResourceRequirements{
+						Requests: v1.ResourceList{
+							v1.ResourceStorage: resource.MustParse("5Gi"),
+						},
+					},
+				},
+			}
+
+			core.Instance().DeletePersistentVolumeClaim(pvcName, appNs)
+			time.Sleep(30 * time.Second)
+			_, err := core.Instance().CreatePersistentVolumeClaim(pvcObj)
+			if err != nil {
+				log.Errorf("Error creating pvc[%v], error[%v]", pvcName, err)
+			}
+			return err
+		}
+
+		sc := "phani"
+		ns := "testns"
+		pvc := "pvcwithtop"
+		err := createSC(sc)
+		log.FailOnError(err, fmt.Sprintf("Failed to create SC, error[%v]", err))
+		defer func() {
+			storage.Instance().DeleteStorageClass(sc)
+		}()
+		err = createNS(ns)
+		log.FailOnError(err, fmt.Sprintf("Failed to create NS, error[%v]", err))
+		err = createPVC(pvc, sc, ns)
+		log.FailOnError(err, fmt.Sprintf("Failed to create PVC, error[%v]", err))
+		defer func() {
+			core.Instance().DeletePersistentVolumeClaim(pvc, ns)
+			core.Instance().DeleteNamespace(ns)
+		}()
+
+		time.Sleep(10 * time.Second)
+		pvc1, err := core.Instance().GetPersistentVolumeClaim(pvc, ns)
+		log.InfoD("PVC pvc[%v] err[%v]", pvc1, err)
+		err = Inst().S.WaitForSinglePVCToBound(pvc, ns, 3)
+		log.FailOnError(err, fmt.Sprintf("error validating pvc"))
+	})
+	JustAfterEach(func() {
+		log.Infof("In Teardown")
+		defer EndTorpedoTest()
+	})
+
+})
+
+var _ = Describe("{FBDATopologyCreateTestNoRegion}", func() {
+	JustBeforeEach(func() {
+		log.Infof("Starting Torpedo tests ")
+		StartTorpedoTest("FBDATopologyCreateTestNoRegion",
+			"Try Creating FBDA pvcs using various topology options", nil, 0)
+	})
+	itLog := "FBDATopologyCreateTestNoRegion"
+	It(itLog, func() {
+		createSC := func(scName string) error {
```

Comment:
we have a function now to create SC , https://github.com/portworx/torpedo/pull/2495/files,  

func CreateFlashStorageClass(scName string,
	scType string,
	ReclaimPolicy v1.PersistentVolumeReclaimPolicy,
	params map[string]string,
	MountOptions []string,
	AllowVolumeExpansion *bool,
	VolumeBinding storageapi.VolumeBindingMode,
	AllowedTopologies map[string][]string) error {

Reply:
Done